src/components/widget/List/RawList.js

```diff
@@ -326,8 +399,9 @@ class RawList extends PureComponent {
             'input-dropdown-list-option-key-on': selectedRow,
           }
         )}
-        onMouseEnter={() => this.handleSwitch(option)}
         onClick={() => this.handleSelect(option)}
+        onMouseEnter={() => this.handleSwitch(option)}
```

Comment:
This has to stay to properly indicate the selected element. We want the selected value to have an indication on the dropdown, but move to the hovered element when mouse is used.

Reply:
Okay, but it needs to be implemented differently then. If you mark the row as selected by simply hovering, the scroll-into-view logic won't work. It will interfere when you are using the arrow keys while the mouse is still in the box

Reply:
Yup. I think using keys should temporarily disable mouse events, until another mouse event is detected.

So it should work like : 
```
user hovers with mouse -> list scrolls ->
user clicks arrow -> disable mouse select (as user is clearly not using it now) ->  list scrolls 
```

some pseudocode : 

```
  navigate = up => {
    (...)
    this.setState({
      disableMouse: true,
      (...)
    });
  };

(...)
        onMouseEnter={() => {
          if (!this.state.disableMouse) {
            this.handleSwitch(option);
          }}}
```

Reply:
Included in https://github.com/metasfresh/metasfresh-webui-frontend/commit/146550d663515bd7d1446d18b07f798609dc54e4